bin/init-cluster

```diff
@@ -74,7 +74,7 @@ echo "K3d cluster configuration exported to ${KUBECONFIG}"
 # Pre-provision PersistentVolumes with RWX accessModes
 
 # Count the number of PVs with RWX accessMode and with the status Available
-AVAILABLE_PV_COUNT=$(kubectl get pv \
+AVAILABLE_PV_COUNT=$(kubectl --kubeconfig="${KUBECONFIG}" get pv \
```

Comment:
Where/when is defined this environment variable?

Reply:
@jmaupetit  Line 70 : 
```
KUBECONFIG=$(k3d kubeconfig write "${K3D_CLUSTER_NAME}")
```

Reply:
Okay, I missed it :sweat_smile: ... sorry for the noise.